packages/kotti-ui/source/kotti-field-toggle/KtFieldToggle.vue

```diff
@@ -37,20 +47,27 @@ export default defineComponent<KottiFieldToggle.PropsInternal>({
 	components: {
 		KtField,
 		ToggleInner,
+		ToggleInnerSuffix,
 	},
 	props: makeProps(KottiFieldToggle.propsSchema),
-	setup(props, { emit }) {
+	setup(props, { emit, slots }) {
 		const field = useField<KottiFieldToggle.Value>({
 			emit,
 			isEmpty: (value) => value !== true,
 			props,
 			supports: KOTTI_FIELD_TOGGLE_SUPPORTS,
 		})
+		const showInnerSuffix = computed(
+			() => Boolean(slots.default) && props.label === null,
+		)
 
 		const { forceUpdate, forceUpdateKey } = useForceUpdate()
 
 		return {
-			field,
+			field: computed(() => ({
```

Comment:
Would give it a different name to the computed property as there is already `const field` in the file